sendme: Adapt error message for Debian

For Debian users installing the backfire-dkms package should be enough to get
the kernel module.

Forwarded: not-needed
---
 src/backfire/sendme.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/src/backfire/sendme.c
+++ b/src/backfire/sendme.c
@@ -220,9 +220,9 @@
 				"try 'modprobe backfire'\n"
 				"If the module backfire can't be loaded, "
 				"it may need to be built first.\n"
-				"Execute 'cd src/backfire; make' in the "
-				"rt-tests directory (requires rt-tests\n"
-				"sources and kernel-devel package).\n");
+				"Try installing the backfire-dkms package "
+				"and kernel headers corresponding to your "
+				"kernel.\n");
 		return 1;
 	}
 	fl.l_type = F_WRLCK;
